terms.rdf

```diff
@@ -517,6 +517,19 @@
         <rdfs:range rdf:resource="http://www.w3.org/2000/01/rdf-schema#Literal"/>
     </rdf:Property>
 
+    <!--
+        Functional properties
+    -->
+    <rdf:Property rdf:about="http://cldf.clld.org/v1.0/terms.rdf#functionalEquivalentsSet" dc:type="functional-property">
+        <rdfs:comment rdf:parseType="Literal">
+            <p>A functional equivalents set if a group of strings from different languages that express similar function.
+                In practice this is an arbitrary name binding together a group of forms from different languages.
+            </p>
+        </rdfs:comment>
+        <csvw:name>"FunctionalEquivalentsSet"</csvw:name>
```

Comment:
We typically follow the (admittedly weird) camel-case plus `_` convention. So it should be `Functional_Equivalent_Set`.